user-interface/src/data-verification/ConsolidationOrderAccordion.tsx

```diff
@@ -43,21 +90,132 @@ export interface ConsolidationOrderAccordionProps {
 
 export function ConsolidationOrderAccordion(props: ConsolidationOrderAccordionProps) {
   const { hidden, statusType, orderType, officesList, expandedId } = props;
-  const caseTable = useRef<OrderTableImperative>(null);
 
-  const confirmationModalRef = useRef<ConfirmationModalImperative>(null);
+  //////////////////// REFS ///////////////////
```

Comment:
I'm not necessarily opposed to this. But could we make them a bit more uniform, like using a block comment or something?
```suggestion
  /***** REFS *****/
```

Reply:
Although I do wonder if there's a way we could instead enforce grouping these things via linting.

Reply:
I was trying to make them stick out when scrolling quickly through the code.  I can change to a block comment though, that's fine.

Reply:
Actually I do have an argument against this:  Using block comments makes things difficult or annoying when you actually want to use block comments to comment something out for a test or whatever, and you don't realize that there is another block comment within it.  Then you have to spend a few minutes figuring out why everything is broken.

I'd rather reserve block comments for rare occasions when you are purposefully trying to block out a large portion of the code.

Reply:
We could do something like //================== REFS ==================